composer.json

```diff
@@ -14,8 +14,8 @@
     },
     "require": {
         "php": ">=5.5",
-        "zendframework/zend-stdlib": "~2.5",
-        "zendframework/zend-servicemanager": "~2.5"
+        "zendframework/zend-stdlib": "dev-develop as 2.7.0",
```

Comment:
Use `dev-develop as 2.8.0` if a stable release fails (develop branch has the SM/EM refactors on that component, but it's possible they won't affect this).